### PR TITLE
Update creating-plugin.rst

### PR DESCRIPTION
**Pull request by vcraescu**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| License         | MIT |

### CODE REVIEW

File: docs/plugins/creating-plugin.rst [hunk 1]
```diff
@@ -65,7 +65,7 @@ to ensure your plugin's extraordinary quality.
 Besides the way you are creating plugins (based on our skeleton or on your own), there are a few naming conventions that should be followed:
 
  * Bundle class must have a `Plugin` suffix instead of `Bundle` in its name (e.g. InvoicePlugin).
```
Comment (pamil) [inline docs/plugins/creating-plugin.rst]: Can you also update these ones (and the ones in 73rd line)? :)

**Comment by vcraescu** [comment]
done.

**Comment by pamil** [comment]
Thanks @vcraescu! 🎉

**Comment by vcraescu** [comment]
uw :)

**Pull request merged**